dashboard/app/views/levels/_dialog.html.haml

```diff
@@ -40,9 +40,19 @@
   #success-dialogcontent
     .modal-content
       %p.dialog-title
-        =t('multi.correct_title')
+        - if data['options'].try(:[], 'success_title')
```

Comment:
Nit: This could be made a little cleaner if you always set `@hash[:options] = {}` in `ContentDSL#initialize`.  Then you can assume `data['options']` exists here and on lines 50-51.